dkron/ui.go

```diff
@@ -58,11 +61,12 @@ func (h *HTTPTransport) UI(r *gin.RouterGroup) {
 				ln = l.Name
 			}
 			ctx.HTML(http.StatusOK, "index.html", gin.H{
-				"DKRON_API_URL":         fmt.Sprintf("/%s", apiPathPrefix),
-				"DKRON_LEADER":          ln,
-				"DKRON_TOTAL_JOBS":      totalJobs,
-				"DKRON_FAILED_JOBS":     failedJobs,
-				"DKRON_SUCCESSFUL_JOBS": successfulJobs,
+				"DKRON_API_URL":         	fmt.Sprintf("/%s", apiPathPrefix),
+				"DKRON_LEADER":          	ln,
+				"DKRON_TOTAL_JOBS":      	totalJobs,
+				"DKRON_FAILED_JOBS":     	failedJobs,
+				"DKRON_PRISTINE_JOBS":   	pristineJobs,
```

Comment:
ui/public/index.html also needs to be updated for the value to show up in the dashboard.
You don't have this file in your fork - pull from the master on distribworks/dkron to get that file.

Reply:
Done